project/src/test/java/com/google/sps/servlets/QueryServletTest.java

```diff
@@ -99,4 +116,14 @@ private int getPlacesAmountInResponse() {
         .fromJson(responseStringWriter.getBuffer().toString(), JsonArray.class)
         .size();
   }
+
+  // Set the parameters for the mocked http request. Most parameter are contant valid values, and
+  // the cuisines parameter is supplied by the calling function.
+  private void setRequestParameters(String cuisines) {
```

Comment:
Suggestion: rename to `initializeRequestParameters`. Seems clearer to me as you don't really set anything..

Reply:
Great idea, thanks